apps/core-app/src/pages/Members.tsx

```diff
@@ -197,13 +197,15 @@ export function Members() {
       actions={[
         <StyledButtonLink
           href={`/molochv3/${daochain}/${daoid}/new-proposal?formLego=ISSUE`}
+          key="1"
```

Comment:
I could be wrong about this, but if we use '1' or '2' as keys then react might confuse them with other lists with keys that have the same name, which could cause perf and state problems. 

Read here: 
https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/
https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
this one isn't inside a map - but i'll switch away from '1', '2' here.

Reply:
actually not going to put those in an array - just wrap in a fragment and send a single node through. feels better.